apps/src/templates/curriculumCatalog/CurriculumCatalogCard.jsx

```diff
@@ -213,14 +219,26 @@ const CustomizableCurriculumCatalogCard = ({
                 : style.buttonsContainer_notEnglish
             )}
           >
-            <Button
-              __useDeprecatedTag
-              color={Button.ButtonColor.neutralDark}
-              type="button"
-              href={pathToCourse}
-              aria-label={quickViewButtonDescription}
-              text={quickViewButtonText}
-            />
+            {!!DCDO.get('quick-view', false) ? (
+              <Button
+                color={Button.ButtonColor.neutralDark}
+                type="button"
+                onClick={handleQuickView}
+                aria-label={quickViewButtonDescription}
+                text={'Quick View'}
```

Comment:
Can you use an i18n string to make sure 'Quick view' is translatable please instead of hard-coded?

Reply:
Nit: I think since the previous text was "Learn more", we should match the capitalization pattern and make it "Quick view" rather than "Quick View"